obstacles:
  config_path: 'config/obstacles.json'

move_l:
  speed: 0.2

  resolution_m: 0.005
  min_resolution_m_offset: -0.005
  max_resolution_m_offset: 0.005

  jump_threshold: 2.0
  min_jump_threshold_offset: -2.0
  max_jump_threshold_offset: 2.0

move_j:
  speed: 0.05  # todo: set it. With `set_max_velocity_scaling_factor()`?

open_gripper:
  width: 0.08
  speed: 0.4

close_gripper:
  # these params should work for both
  #   - no-object present -> full close
  #   - object present
  # (a bit hacky) to cope with both with a margin, one can play with the `width` and `epsilon_`:
  #   - for a Xcm object: width=X/2 and epsilon=X/2+margin
  width: 0.03
  epsilon_inner: 0.04  # max tolerated deviation when the actual grasped width is smaller than the commanded grasp width
  epsilon_outer: 0.04  # max tolerated deviation when the actual grasped width is larger than the commanded grasp width

  speed: 0.1  # Closing speed [m/s]
  force: 20  # Grasping (continuous) force [N]

my_move:
  n_trials_plan_l: 10
